packages/api/amplify_api/example/integration_test/util.dart

```diff
@@ -165,50 +165,28 @@ Future<Post> addPostAndBlog(
   return data;
 }
 
-Future<GraphQLRequest<T>> authorizeRequestForUserPools<T>(
-    GraphQLRequest<T> request) async {
-  // TODO(ragingsquirrel3): convert to authorizationType parameter when implemented.
-  final authSession =
-      await Amplify.Auth.fetchAuthSession() as CognitoAuthSession;
-  final accessToken = authSession.userPoolTokens?.accessToken.raw;
-  if (accessToken == null) {
-    throw const AmplifyException(
-      'Could not get access token from cognito.',
-      recoverySuggestion: 'Ensure test user signed in.',
-    );
-  }
-  final headers = {AWSHeaders.authorization: accessToken};
-
+GraphQLRequest<T> authorizeRequestForUserPools<T>(GraphQLRequest<T> request) {
   return GraphQLRequest<T>(
     document: request.document,
     variables: request.variables,
     modelType: request.modelType,
     decodePath: request.decodePath,
-    headers: headers,
+    authorizationMode: APIAuthorizationType.userPools,
```

Comment:
Is APIAuthorizationType the correct name? I just realized we call it `mode` almost everywhere else

Reply:
Are suggesting maybe renaming the enum to `APIAuthorizationMode`?  That might be better as I believe usage in gql request is the most prominent (kinda only) place we are exposing publicly so might be better if matched. On the other hand, I think "Type" is ok and does not have to match the property name in this case.

Reply:
I just pushed a commit that renamed this everywhere and re-generated the .g.dart files that get affected.

Reply:
and reverted this